### PR TITLE
Add dns_domain and max_node_unavailable default values

### PR DESCRIPTION
**Pull request by qynonyq**

### CODE REVIEW

**Review by schirevko — CHANGES_REQUESTED** [review]
Please, update example according to readme: https://github.com/vk-cs/terraform-provider-vkcs/blob/master/examples/README.md
There is new naming for examples, e.g example for creating cluster should be named as "vkcs-kubernetes-cluster-create": https://github.com/vk-cs/terraform-provider-vkcs/tree/master/examples/vkcs-kubernetes-cluster-create
This will enable canary test system to automatically run examples and ensure that they are working.

File: examples/vkcs-kubernetes-cluster-create/provider.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+provider "vkcs" {
+  username   = "user@mail.ru"
+  password   = "password"
+  project_id = "project_id"
+}
```
[the remark this answers is not in the document]
Comment (qynonyq) [reply examples/vkcs-kubernetes-cluster-create/provider.tf]: removed

File: examples/vkcs-kubernetes-node-group-create/base.tf [hunk 2]
```diff
@@ -1,31 +1,8 @@
...
-
-data "vkcs_compute_flavor" "k8s" {
-  name = "Standard-2-4-50"
+terraform {
+  required_providers {
+    vkcs = {
+      source  = "vk-cs/vkcs"
+      version = "~> 0.1.0"
+    }
+  }
```
[the remark this answers is not in the document]
Comment (qynonyq) [reply examples/vkcs-kubernetes-node-group-create/base.tf]: removed

File: examples/vkcs-kubernetes-node-group-create/main.tf [hunk 3]
```diff
@@ -1,31 +1,11 @@
...
-  network_id          = vkcs_networking_network.k8s.id
-  subnet_id           = vkcs_networking_subnet.k8s-subnetwork.id
-  floating_ip_enabled = true
-  availability_zone   = "MS1"
-  insecure_registries = ["1.2.3.4"]
-}
-
 resource "vkcs_kubernetes_node_group" "default_ng" {
-    cluster_id = vkcs_kubernetes_cluster.k8s-cluster.id
+    cluster_id = "your_cluster_id"
```
[the remark this answers is not in the document]
Comment (qynonyq) [reply examples/vkcs-kubernetes-node-group-create/main.tf]: fixed

**Review by schirevko — APPROVED** [review]

**Pull request merged**